src/gpacmp4/box_code_base.c

```diff
@@ -1311,7 +1311,7 @@ GF_Err gnrm_Write(GF_Box *s, GF_BitStream *bs)
 	GF_Err e;
```

Comment:
Any spelling correction in gpacmp4 is quite useless, as that's an external library. It'll be gone next time we update gpacmp4